modules/virt-disable-individual-bootsource-update.adoc

```diff
@@ -5,13 +5,13 @@
 
 :_content-type: PROCEDURE
 [id="virt-disable-individual-bootsource-update_{context}"]
-= Disabling an automatic update for a custom boot source
+= Disabling an automatic update for a single boot source
 
-You can reduce the number of logs on disconnected environments or reduce resource usage by disabling the automatic imports and updates for an individual custom boot source.
+You can reduce the number of logs on disconnected environments or reduce resource usage by disabling the automatic imports and updates for a single boot source that has not been customized.
 
 .Procedure
 
-* Edit the `HyperConverged` custom resource (CR) and disable updates for a custom boot source by setting the `dataimportcrontemplate.kubevirt.io/enable` annotation in the CR to `false`.
+* Edit the `HyperConverged` custom resource (CR) and disable updates for a single boot source by setting the `dataimportcrontemplate.kubevirt.io/enable` annotation in the CR to `false`.
```

Comment:
Pre-defined resource are (by default) not listed in the `spec.dataImportCronTemplates` field.

In order to disable them (or to modified them), the user must ***add*** the specific source to the list, and edit it.

For reference, all the currently active boot resources are listed in the `status.dataImportCronTemplates` field. So the user can copy the boot-source object from the status filed to the spec field, and edit it there.

Reply:
Added this information.